package.json

```diff
@@ -12,5 +12,14 @@
   "devDependencies": {
     "grunt": "^1.0.1",
     "shelljs": "^0.7.8"
-  }
+  },
+  "files": [
+    "version.txt",
+    "LICENSE",
+    "CHANGELOG.md",
+    "README.md",
+    "framework/Manifest.json",
+    "framework/source",
+    "tool/data/cldr"
+  ]
```

Comment:
> "Certain files are always included, regardless of settings:
> package.json
> README
> CHANGES / CHANGELOG / HISTORY
> LICENSE / LICENCE
> NOTICE
> The file in the "main" field
> README, CHANGES, LICENSE & NOTICE can have any case and extension.

see https://docs.npmjs.com/files/package.json#files

Reply:
I know. I just wanted to document the Inklusion.